syntax = "proto3";

option java_package = "com.huasheng.sa.quant.open.sdk.protobuf.hq.notify";
option java_outer_classname = "TradeTickerNotifyProto";
message TradeTickerV2Notify {
	string stockCode = 1;
	int64 tradeTime = 2;
	string price = 3;
	int64 volume = 4;//交易量（不包含单位）
	string side = 5; // A：买盘，V：卖盘，O：平盘
	string trdType = 6; // 交易类型
	int64 timestamp = 7; // 时间戳（取代tradeTime）
	int64 sourceTime = 8;//供应商时间戳
	int64 sourceRecvTime = 9;//接收端收到时间戳
	int64 sourcePushTime = 10;//推送端推送时间戳
	string	sourceTrdType = 11; //tick类型（用于过滤美股tick）
	int32 mktTmType = 12; //1:盘中 -1:盘前 -2:盘后
}
